<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HVTNDAL0215" ignored="0" name="Test" tests="4" failures="2" timestamp="02 Nov 2022 09:44:54 GMT" time="0.911" errors="0">
  <testcase name="case1" time="0.098" classname="allreTestCases.Testingg">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at allreTestCases.Testingg.case1(Testingg.java:41)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- case1 -->
  <testcase name="case2" time="0.005" classname="allreTestCases.Testingg">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at allreTestCases.Testingg.case2(Testingg.java:50)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- case2 -->
  <testcase name="case3" time="0.009" classname="allreTestCases.Testingg"/>
  <testcase name="case4" time="0.004" classname="allreTestCases.Testingg"/>
</testsuite> <!-- Test -->
